mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -107,6 +107,9 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 		ARG 1 entity
 		ARG 2 hand
 	METHOD method_7288 findRespawnPosition (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;Z)Ljava/util/Optional;
+		ARG 0 worldView
```

Comment:
This should be either `world` or `view`. (Which one did we decide to use for `WorldView`, or is it mixed?)

Reply:
preferably `world`. `view` is more used for `BlockView` etc.

Reply:
```suggestion
		ARG 0 world
```